frontend/src/api/collimator/hooks/tasks/useDeleteTask.ts

```diff
@@ -4,19 +4,19 @@ import { tasksControllerRemoveV0 } from "../../generated/endpoints/tasks/tasks";
 import { useRevalidateTaskList } from "./useRevalidateTaskList";
 import { useAuthenticationOptions } from "../authentication/useAuthenticationOptions";
 
-type DeleteUserType = (id: number) => Promise<DeletedTask>;
+type DeleteTaskType = (id: number) => Promise<DeletedTask>;
```

Comment:
Minor nitpick: unless this is already the convention in the project, I would avoid using `type` in the name, since this information is included in the type.

```suggestion
type DeleteTask = (id: number) => Promise<DeletedTask>;
```

Reply:
Good point, but copied over from some other files where indeed it's more or less the convention.
I would also be wary of having something called `DeleteTask` as it seems more of an action than a  thing.